x-pack/plugins/ml/public/application/services/ml_api_service/index.ts

```diff
@@ -368,6 +368,14 @@ export function mlApiProvider(httpService: HttpService) {
       });
     },
 
+    deleteForecast({ jobId, forecastId }: { jobId: string; forecastId: string }) {
+      return httpService.http<any>({
```

Comment:
It would be good to add the correct return type here

Reply:
Updated in d29de18790177bc0c0d512a43845f70518ab6dc7